src/styles/blocks/catalog.scss

```
@@ -0,0 +1,139 @@
+.catalog {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: center;
+  column-gap: 46px;
+  row-gap: 48px;
```

should be vice versa
```suggestion
  column-gap: 48px;
  row-gap: 46px;
```